Apply stylesheet changes and additions from newest SUL styles

jgreben opened the pull request:
Fixes #16

jgreben committed:
Apply stylesheet changes and additions from newest SUL styles, per JV

jgreben committed:
Apply scss-linter changes

jgreben committed:
Change h1, h2 font for media sizes

jvine commented on app/assets/stylesheets/base.scss:
```diff
@@ -10,14 +10,35 @@ a {
   color: $sul-h1-font-color;
-  font-size: 2.3em;
+  font-size: 2.1em;
   letter-spacing: -0.05em;
```
Kill letter-spacing - it's only relevant to SearchWorks when using deja-vu font.

jvine commented on app/assets/stylesheets/variables.scss:
```diff
@@ -11,7 +11,7 @@ $btn-default-border: $btn-beige-border;
 // Sizes
-$font-size-base: 14px;
+$font-size-base: 15px;
 $line-height-base: 1.428571429;
```
change base line-height:1.6

jvine commented on app/assets/stylesheets/base.scss:
```diff
@@ -10,14 +10,35 @@ a {
+  font-weight: 100;
+}
+
+@media (min-width: 980px) {
```
I think these media settings are backwards. 

small media / default has this:
h1{font-size:37px;font-size:2.3125rem}  
h2{font-size:26px;font-size:1.625rem;font-weight:100;color:#8c1515}

large media has this:
@media (min-width: 48em){h1{font-size:45px;font-size:2.8125rem}}
@media (min-width: 48em){h2{font-size:31px;font-size:1.9375rem}}

jvine reviewed (commented):
Not seeing the a settings - should they be in this section?
a{text-decoration:none}
a:hover{text-decoration:underline}
a{color:#a26624}

jgreben committed:
Move anchor tag style to base.scss, remove unused form.scss for now, and increase line-height base

jgreben commented:
@jvine the `a` settings were in the top-nav css and the form css. I removed the form.css because I don't think we need it (yet) and moved the `a` settings to base.css. I fixed the rest of the things you mentioned. Here is the HTML for the large media view:
```<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta name="description" content="Stanford University Libraries - My borrowing account" />
    <!-- Mobile viewport optimization h5bp.com/ad -->
    <meta name="HandheldFriendly" content="True">
    <meta name="viewport" content="width=device-width,initial-scale=1.0">
    <!-- Mobile IE allows us to activate ClearType technology for smoothing fonts for easy reading -->
    <!--[if IEMobile]> <meta http-equiv="cleartype" content="on"> <![endif]-->

    <title>My borrowing account</title>
    <link rel="shortcut icon" type="image/x-icon" href="/assets/favicon-8a4c41227945e5f20503f62517defcfb2ef88f578e222838318c704472d483b0.ico" />
    <link rel="stylesheet" media="all" href="/assets/application.self-847108eb5c2be131a9ae49c26ab36bbc2d5c304bb76fa07606421715068029df.css?body=1" data-turbolinks-track="true" />
    <link href="https://www.stanford.edu/su-identity/css/su-identity.css" rel="stylesheet">
    <link rel="stylesheet" media="screen" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" />
    <script src="/assets/jquery.self-bd7ddd393353a8d2480a622e80342adf488fb6006d667e8b42e4c0073393abee.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/jquery_ujs.self-784a997f6726036b1993eb2217c9cb558e1cbb801c6da88105588c56f13b466a.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/bootstrap.self-b38817c3e3a3049abb1fc08dd6ae448b23330f8453226efdb074710209474f75.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/turbolinks.self-5da219edf6499ec4a029db45e1d9bd1675b067816a7645daf247de4155433350.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/application.self-f8806224e027f3e3f0138ea9ce99319e298dfdb323304d1f1be6eae8e8c74724.js?body=1" data-turbolinks-track="true"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.26.2/js/jquery.tablesorter.js"></script>
    <meta name="csrf-param" content="authenticity_token" />
<meta name="csrf-token" content="5a28Sb/pKyXm59u7F2lIzgDgaTCtOTwsSDlojOyRcwmos7cN4jQVd4IVno6WZTc4Acyl7kTw5MUaWWGNlky0Zg==" />
    
      <!--[if lt IE 9]> <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script> <![endif]-->
      <!--[if IE 8]> <link rel="stylesheet" type="text/css" href="https://www.stanford.edu/su-identity/css/ie/ie8.css" /> <![endif]-->
      <!--[if IE 7]> <link rel="stylesheet" type="text/css" href="https://www.stanford.edu/su-identity/css/ie/ie7.css" /> <![endif]-->
  </head>

  <body>
    <div id="su-wrap"> <!-- #su-wrap start -->
      <div id="su-content"> <!-- #su-content start -->
        <div id="topnav-container" class="container-fluid">
          <div id="topnav-container">
  <header id="topnav" class="header-logo" role="banner">
    <a href="http://library.stanford.edu">
    <span class="sr-only">Stanford University Libraries</span>
</a>
    <div class="header-links" id="login_id">
        <a href="/webauth/login?referrer=http%3A%2F%2Flocalhost%3A3000%2F">Login</a>
      <a href="http://searchworks.stanford.edu/">SearchWorks</a>
      <a target="_blank" rel="noopener noreferrer" href="http://searchworks.stanford.edu/feedback">Feedback</a>
    </div>
  </header>
</div>

<div class="topnav-divider">
  <img class="logo" alt="" src="/assets/logo-8927b2b7311fb40a2c242041bf22391759febf39e837bccd7d8c2d71327f714e.svg" />
</div>
          <section id="main-container" role="main"> 
            <div class="home-page-section">
  <h1>My borrowing account</h1>
</div>
          </section>
        </div>
      </div>
    </div>

      <footer role="“contentinfo”">
  <div id="sul-footer-container">
    <div id="sul-footer">
      <div id="sul-footer-img" class="span2">
        <img alt="" src="/assets/sul-logo-stacked-665eb0a32fce13aea9d1239388277bdc77c70428f3c86b33847f00152020bc0c.svg" />
      </div>
      <div id="sul-footer-links" class="span2">
        <ul>
          <li><a href="http://library.stanford.edu">Stanford University Libraries</a></li>
          <li><a href="http://library.stanford.edu/hours">Hours &amp; locations</a></li>
          <li> <a href="http://library.stanford.edu/ask">Ask us</a></li>
          <li> <a href="http://library.stanford.edu/opt-out">Opt out of analytics</a></li>
        </ul>
      </div>
    </div>
  </div>
</footer>

<!-- Global footer snippet start -->
<div id="global-footer">
  <div class="container">
    <div class="row">
      <div id="bottom-logo" class="span2">
        <a href="http://www.stanford.edu">
          <img src="https://www.stanford.edu/su-identity/images/footer-stanford-logo@2x.png" alt="Stanford University" width="105" height="49"/>
        </a>
      </div>
      <!-- #bottom-logo end -->
      <div id="bottom-text" class="span10">
        <ul>
          <li class="home"><a href="http://www.stanford.edu">SU Home</a></li>
          <li class="maps alt"><a href="http://visit.stanford.edu/plan/maps.html">Maps &amp; Directions</a></li>
          <li class="search-stanford"><a href="http://www.stanford.edu/search/">Search Stanford</a></li>
          <li class="terms alt"><a href="http://www.stanford.edu/site/terms.html">Terms of Use</a></li>
          <li class="emergency-info"><a href="http://emergency.stanford.edu">Emergency Info</a></li>
        </ul>
      </div> <!-- .bottom-text end -->
      <div class="clear"></div>
      <p class="copyright vcard">&copy; <span class="fn org">Stanford University</span>, <span class="adr"> <span class="locality">Stanford</span>, <span class="region">California</span> <span class="postal-code">94305</span></span>.&nbsp;&nbsp;
      <span id="copyright-complaint"><a href="http://www.stanford.edu/group/security/dmca.html" class="copyright-complaint" >Copyright Complaints</a></span>
      </p>
    </div> <!-- .row end -->
  </div> <!-- .container end -->
</div> <!-- global-footer end -->
<!-- Global footer snippet end -->
  </body>
</html>
```

Here it the HML for the small media view:
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta name="description" content="Stanford University Libraries - My borrowing account" />
    <!-- Mobile viewport optimization h5bp.com/ad -->
    <meta name="HandheldFriendly" content="True">
    <meta name="viewport" content="width=device-width,initial-scale=1.0">
    <!-- Mobile IE allows us to activate ClearType technology for smoothing fonts for easy reading -->
    <!--[if IEMobile]> <meta http-equiv="cleartype" content="on"> <![endif]-->

    <title>My borrowing account</title>
    <link rel="shortcut icon" type="image/x-icon" href="/assets/favicon-8a4c41227945e5f20503f62517defcfb2ef88f578e222838318c704472d483b0.ico" />
    <link rel="stylesheet" media="all" href="/assets/application.self-847108eb5c2be131a9ae49c26ab36bbc2d5c304bb76fa07606421715068029df.css?body=1" data-turbolinks-track="true" />
    <link href="https://www.stanford.edu/su-identity/css/su-identity.css" rel="stylesheet">
    <link rel="stylesheet" media="screen" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" />
    <script src="/assets/jquery.self-bd7ddd393353a8d2480a622e80342adf488fb6006d667e8b42e4c0073393abee.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/jquery_ujs.self-784a997f6726036b1993eb2217c9cb558e1cbb801c6da88105588c56f13b466a.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/bootstrap.self-b38817c3e3a3049abb1fc08dd6ae448b23330f8453226efdb074710209474f75.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/turbolinks.self-5da219edf6499ec4a029db45e1d9bd1675b067816a7645daf247de4155433350.js?body=1" data-turbolinks-track="true"></script>
<script src="/assets/application.self-f8806224e027f3e3f0138ea9ce99319e298dfdb323304d1f1be6eae8e8c74724.js?body=1" data-turbolinks-track="true"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.tablesorter/2.26.2/js/jquery.tablesorter.js"></script>
    <meta name="csrf-param" content="authenticity_token" />
<meta name="csrf-token" content="gBaREOlDzLAMOlm2wqXGzreSWJpbSW5R1m0DUk1q3F3NCJpUtJ7y4mjIHINDqbk4tr6URLKAtriEDQpTN7cbMg==" />
    
      <!--[if lt IE 9]> <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script> <![endif]-->
      <!--[if IE 8]> <link rel="stylesheet" type="text/css" href="https://www.stanford.edu/su-identity/css/ie/ie8.css" /> <![endif]-->
      <!--[if IE 7]> <link rel="stylesheet" type="text/css" href="https://www.stanford.edu/su-identity/css/ie/ie7.css" /> <![endif]-->
  </head>

  <body>
    <div id="su-wrap"> <!-- #su-wrap start -->
      <div id="su-content"> <!-- #su-content start -->
        <div id="topnav-container" class="container-fluid">
          <div id="topnav-container">
  <header id="topnav" class="header-logo" role="banner">
    <a href="http://library.stanford.edu">
    <span class="sr-only">Stanford University Libraries</span>
</a>
    <div class="header-links" id="login_id">
        <a href="/webauth/login?referrer=http%3A%2F%2Flocalhost%3A3000%2F">Login</a>
      <a href="http://searchworks.stanford.edu/">SearchWorks</a>
      <a target="_blank" rel="noopener noreferrer" href="http://searchworks.stanford.edu/feedback">Feedback</a>
    </div>

  </header>
</div>

<div class="topnav-divider">
  <img class="logo" alt="" src="/assets/logo-8927b2b7311fb40a2c242041bf22391759febf39e837bccd7d8c2d71327f714e.svg" />
</div>

          <section id="main-container" role="main">
            
            <div class="home-page-section">
  <h1>My borrowing account</h1>
</div>

          </section>
        </div>
      </div>
    </div>

      <footer role="“contentinfo”">
  <div id="sul-footer-container">
    <div id="sul-footer">
      <div id="sul-footer-img" class="span2">
        <img alt="" src="/assets/sul-logo-stacked-665eb0a32fce13aea9d1239388277bdc77c70428f3c86b33847f00152020bc0c.svg" />
      </div>
      <div id="sul-footer-links" class="span2">
        <ul>
          <li><a href="http://library.stanford.edu">Stanford University Libraries</a></li>
          <li><a href="http://library.stanford.edu/hours">Hours &amp; locations</a></li>
          <li> <a href="http://library.stanford.edu/ask">Ask us</a></li>
          <li> <a href="http://library.stanford.edu/opt-out">Opt out of analytics</a></li>
        </ul>
      </div>
    </div>
  </div>
</footer>

<!-- Global footer snippet start -->
<div id="global-footer">
  <div class="container">
    <div class="row">
      <div id="bottom-logo" class="span2">
        <a href="http://www.stanford.edu">
          <img src="https://www.stanford.edu/su-identity/images/footer-stanford-logo@2x.png" alt="Stanford University" width="105" height="49"/>
        </a>
      </div>
      <!-- #bottom-logo end -->
      <div id="bottom-text" class="span10">
        <ul>
          <li class="home"><a href="http://www.stanford.edu">SU Home</a></li>
          <li class="maps alt"><a href="http://visit.stanford.edu/plan/maps.html">Maps &amp; Directions</a></li>
          <li class="search-stanford"><a href="http://www.stanford.edu/search/">Search Stanford</a></li>
          <li class="terms alt"><a href="http://www.stanford.edu/site/terms.html">Terms of Use</a></li>
          <li class="emergency-info"><a href="http://emergency.stanford.edu">Emergency Info</a></li>
        </ul>
      </div> <!-- .bottom-text end -->
      <div class="clear"></div>
      <p class="copyright vcard">&copy; <span class="fn org">Stanford University</span>, <span class="adr"> <span class="locality">Stanford</span>, <span class="region">California</span> <span class="postal-code">94305</span></span>.&nbsp;&nbsp;
      <span id="copyright-complaint"><a href="http://www.stanford.edu/group/security/dmca.html" class="copyright-complaint" >Copyright Complaints</a></span>
      </p>
    </div> <!-- .row end -->
  </div> <!-- .container end -->
</div> <!-- global-footer end -->
<!-- Global footer snippet end -->
  </body>
</html>
```
